root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+              <strong>Please use this date only if you’re certain this date is correct.</strong>') %]
+      </p>
+    <!-- /ko -->
+    <!-- ko if: target() && target().hasJanuaryFirstDate() -->
+      <p>
+        [% l('If you do not know the month or day of release, please leave them empty.
+              January 1st is not often the actual release date.
+              <strong>Please double-check that you have entered the release date correctly.</strong>') %]
+      </p>
+    <!-- /ko -->
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.countryBubble">
+    <p>
+      [% l('The {doc|country} describes where the release was made available.',
+           { doc => { href => doc_link('Release/Country'), target => '_blank' } }) %]
+    </p>
     <p>
-      [% l('The script attribute should be set to the script used for the release title and track titles.') %]
+      If you aren’t sure where it was released, please leave this unset.
     </p>
   </div>
 
+  <div class="bubble" data-bind="bubble: $root.catalogNumberBubble">
+    <!-- ko ifnot: target() && catNoLooksLikeASIN(target().catalogNumber()) -->
+      <p>
+        [% l('The {doc|catalog number} is assigned by the label and can often be found on the spine or near the barcode on physical media.',
+             { doc => { href => doc_link('Release#Catalogue_number'), target => '_blank' } }) %]
+      </p>
+      <p>
+        If you can’t find a catalog number, please leave this blank.
+      </p>
+    <!-- /ko -->
+    <!-- ko if: target() && catNoLooksLikeASIN(target().catalogNumber()) -->
+      <p>
+        [% l('The catalog number you have entered looks like an Amazon ASIN.') %]
+      </p>
+      <p>
+        [% l('If this is an Amazon ASIN, please do not enter it in the catalog number field, but add it as a URL relationship later.') %]
+      </p>
+    <!-- /ko -->
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.labelBubble">
+    <!-- ko with: target() && target().label() -->
+      <!-- ko if: $data.gid -->
+        <p data-bind="html: $data.selectionMessage()"></p>
+      <!-- /ko -->
+      <!-- ko ifnot: $data.gid -->
+        <p>
+          [% l('The {doc|label} is the imprint associated with the release.
+                Please also see the guidelines about {style|special purpose labels}.',
+               { doc => { href => doc_link('Label'), target => '_blank' },
+                 style => { href => doc_link('Style/Unknown_and_untitled/Special_purpose_label'), target => '_blank' } }) %]
+        </p>
+      <!-- /ko -->
+    <!-- /ko -->
+  </div>
+
   <div class="bubble" data-bind="bubble: $root.barcodeBubble">
     <p>
-      [% l('Please enter the barcode of the release you are entering, see <a href="{url}" target="_blank">Barcode</a> for more information.',
-           { url => doc_link('Barcode') }) %]
+      [% l('The {doc|barcode} is the alphanumeric identifier (e.g. UPC or EAN) associated with the release.',
```

Comment:
I'm not very happy with the uses of "alphanumeric" and "e.g." in this sentence.

Reply:
Instead of "e.g." use "such as" then? For "alphanumeric" - it's not, is it? `is_valid_barcode` checks `/^[0-9]+$/` so it's just numeric.